_health-care/_js/components/HealthCareApp.jsx

```diff
@@ -220,16 +219,39 @@ class HealthCareApp extends React.Component {
   getExternalData(applicationData, statePath) {
     switch (statePath[0]) {
       case 'financial-assessment':
+        if (statePath[1] === 'annual-income') {
```

Comment:
I'll make a change with the addition of a check of the `propertyPath` to conditionally update.